build/syncset.go

```diff
@@ -388,27 +386,43 @@ func createSelectorSyncSet(resources []runtime.RawExtension) *hivev1.SelectorSyn
 				ResourceApplyMode: hivev1.SyncResourceApplyMode,
 				Resources:         resources,
 			},
-			ClusterDeploymentSelector: metav1.LabelSelector{
-				MatchLabels: map[string]string{
-					"api.openshift.com/managed": "true",
-				},
-			},
+			ClusterDeploymentSelector: selector,
 		},
 	}
 }
+
+type SyncSetResourcesByLabelSelector struct {
+	entries []SyncSetResource
+}
+
+type SyncSetResource struct {
+	labelSelector metav1.LabelSelector
+	objects       []runtime.RawExtension
+}
```

Comment:
For future us: "Why can't we just use a `map[metav1.LabelSelector]someOtherType`" ??? Maybe a short statement as to what these to structs are for/doing?

Reply:
done